aries_vcx/tests/pgwallet/wallet_plugin_loader.rs

```diff
@@ -5,12 +5,52 @@ use serde_json::Value;
 
 use aries_vcx::indy::ErrorCode;
 
-use crate::pgwallet::dyn_lib::load_lib;
 use crate::pgwallet::constants::POSTGRES_ADDITIONAL_INITIALIZER;
 use crate::pgwallet::constants::DEFAULT_POSTGRES_PLUGIN_PATH;
 use crate::pgwallet::constants::DEFAULT_POSTGRES_PLUGIN_INITIALIZER;
 
-pub fn load_storage_library(library: &str, initializer: &str) -> Result<libloading::Library, String> {
+#[cfg(all(unix, test))]
+pub fn load_lib(library: &str) -> libloading::Result<libloading::Library> {
+    libloading::os::unix::Library::open(Some(library), ::libc::RTLD_NOW | ::libc::RTLD_NODELETE)
+        .map(libloading::Library::from)
+}
+
+#[cfg(any(not(unix), not(test)))]
+pub fn load_lib(library: &str) -> libloading::Result<libloading::Library> {
+    libloading::Library::new(library)
+}
+
+
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct PluginInitConfig {
+    pub storage_type: String,
+    // Optional to override default library path. Default value is determined based on value of
+    // xtype and OS
+    pub plugin_library_path: Option<String>,
+    // Optional to override default storage initialization function. Default value is  determined
+    // based on value of xtype and OS
+    pub plugin_init_function: Option<String>,
+    // Wallet storage config for agents wallets
+    pub config: String,
+    // Wallet storage credentials for agents wallets
+    pub credentials: String,
+}
+
+pub fn init_wallet_plugin(plugin_init_config: &PluginInitConfig) -> Result<(), String> {
+    info!("_init_wallet >>> wallet_storage_config:\n{}", serde_json::to_string(plugin_init_config).unwrap());
```

Comment:
incorrect log + newline is not consistent with other logs + do we have log both entry and exit at info level?